test-ts/utils.ts

```diff
@@ -451,3 +472,156 @@ export async function setGovernanceConcurrentProposals(count: number) {
     from: governanceOwner,
   });
 }
+
+export async function getGroupsSafe(
+  defaultStrategy: DefaultStrategy,
+  specificGroupStrategy: SpecificGroupStrategy
+) {
+  const activeGroupsLengthPromise = defaultStrategy.getGroupsLength();
```

Comment:
Because I can request both
defaultStrategy.getGroupsLength()
specificGroupStrategy.getSpecificGroupStrategiesLength()
and await it only when I need it.

It is a bit faster this way